### ISSUE TITLE
Steam freezes the pc while processing Vulkan shaders in the background

### ISSUE THREAD

**Issue by anyoneyun** [ValveSoftware/steam-for-linux#10009]
As the title says, when Steam opens on my machine and background vulkan shaders processing is enabled, it freezes the machine, requiring a hard reboot. If I wait long enough, Steam sometimes crashes. I can provide any log or information. This is happening on Manjaro
My system information are:

```
Computer Information:
Manufacturer: Micro-Star International Co., Ltd.
Model: MPG B550 GAMING PLUS (MS-7C56)
Form Factor: Desktop
No Touch Input Detected
Processor Information:
CPU Vendor: AuthenticAMD
CPU Brand: AMD Ryzen 5 5600X 6-Core Processor
CPU Family: 0x19
CPU Model: 0x21
CPU Stepping: 0x0
CPU Type: 0x0
Speed: 4650 MHz
12 logical processors
6 physical processors
Hyper-threading: Supported
FCMOV: Supported
SSE2: Supported
SSE3: Supported
SSSE3: Supported
SSE4a: Supported
SSE41: Supported
SSE42: Supported
AES: Supported
AVX: Supported
AVX2: Supported
AVX512F: Unsupported
AVX512PF: Unsupported
AVX512ER: Unsupported
AVX512CD: Unsupported
AVX512VNNI: Unsupported
SHA: Supported
CMPXCHG16B: Supported
LAHF/SAHF: Supported
PrefetchW: Unsupported
Operating System Version:
"Manjaro Linux" (64 bit)
Kernel Name: Linux
Kernel Version: 6.4.11-1-MANJARO
X Server Vendor: The X.Org Foundation
X Server Release: 12302000
X Window Manager: KWin
Steam Runtime Version: steam-runtime_0.20230606.51628
Video Card:
Driver: AMD AMD Radeon RX 6600 XT (navi23, LLVM 15.0.7, DRM 3.52, 6.4.11-1-MANJARO)
Driver Version: 4.6 (Compatibility Profile) Mesa 23.1.6
OpenGL Version: 4.6
Desktop Color Depth: 24 bits per pixel
Monitor Refresh Rate: 59 Hz
VendorID: 0x1002
DeviceID: 0x73ff
Revision Not Detected
Number of Monitors: 1
Number of Logical Video Cards: 1
Primary Display Resolution: 3840 x 2160
Desktop Resolution: 3840 x 2160
Primary Display Size: 24.02" x 13.78" (27.68" diag), 61.0cm x 35.0cm (70.3cm diag)
Primary VRAM: 8192 MB
Sound card:
Audio device: USB Mixer
Memory:
RAM: 15915 Mb
VR Hardware:
VR Headset: None detected
Miscellaneous:
UI Language: English
LANG: it_IT.UTF-8
Total Hard Disk Space Available: 479328 MB
Largest Free Hard Disk Block: 262987 MB
Storage:
Number of SSDs: 4
SSD sizes: 1000G,1000G,512G,512G
Number of HDDs: 0
Number of removable drives: 0
```

**Comment by gleizoux** [comment]
Hi

Same here, with RX 6800XT, but not with all games. For example, the OS crashes with ARK, but no with Dirt 2.0 or Rocket League. 
Downgrading vulkan-headers and vulkan-radeon packages solve my problem during one week, but when I wanted to install some app I had to upgrade anothers packages, this broke my vulkan-headers and vulkan-radeon packages. I had to update all files, and now games crash. 

Do you found any solution ?

**Comment by kisak-valve** [comment]
Hello @anyoneyun, mesa 23.1.6 (specifically) has a memory leak which has been fixed in mesa 23.1.7. The fix here is to update mesa when it becomes available through the normal update process for your distro.

Previously noted at https://github.com/ValveSoftware/Fossilize/issues/230.

Closing as not a Steam issue.
